test/cache-interceptor/sqlite-cache-store-tests.js

```diff
@@ -125,3 +125,49 @@ test('SqliteCacheStore maxEntries', async (t) => {
 
   strictEqual(store.size <= 11, true)
 })
+
+test('two writes', async (t) => {
+  if (!hasSqlite) {
+    t.skip()
+    return
+  }
+
+  const SqliteCacheStore = require('../../lib/cache/sqlite-cache-store.js')
+  const sqliteLocation = 'cache-interceptor.sqlite'
+
+  const store = new SqliteCacheStore({
+    location: sqliteLocation,
+    maxCount: 10
+  })
+
+  t.after(async () => {
+    await rm(sqliteLocation)
+  })
+
+  const request = {
+    origin: 'localhost',
+    path: '/',
+    method: 'GET',
+    headers: {}
+  }
+
+  const requestValue = {
+    statusCode: 200,
+    statusMessage: '',
+    rawHeaders: [Buffer.from('1'), Buffer.from('2'), Buffer.from('3')],
```

Comment:
to align w/ the new handler types

```suggestion
    headers: { foo: 'bar' }
```